ui/apps/platform/src/services/APITokensService.js

```diff
@@ -13,6 +13,12 @@ export function fetchAPITokens() {
     }));
 }
 
+export function fetchAllowedRoles() {
+    return axios.get(`${url}/generate/allowed-roles`).then((response) => ({
+        response: response.data,
+    }));
```

Comment:
Nit: In more recent UI service files we have done away with the `response` property wrapper here, and just return the relevant result directly. This lets us avoid the `roles.response.roles` pattern in the previous file as well.

```suggestion
    return axios.get(`${url}/generate/allowed-roles`).then((response) => response.data.roles);
```